Jenkinsfile

```diff
@@ -21,19 +21,35 @@ timestamps {
 
     stage("test") {
       exec('yarn test')
-
-      bedrockBrowsers(
-        prepareTests: {
-          yarnInstall()
-          exec('yarn build')
-        },
-        testDirs: [ 'modules/sample/src/test/ts/**/pass' ],
-        custom: '--config modules/sample/tsconfig.json --customRoutes modules/sample/routes.json --polyfills Promise Symbol'
-      )
     }
+  }
+
+  // Testing
+  stage("bedrock testing") {
+    bedrockRemoteBrowsers(
+      platforms: [
+        [ browser: 'chrome', provider: 'aws', buckets: 2 ],
+        [ browser: 'firefox', provider: 'aws', buckets: 2 ],
+        [ browser: 'edge', provider: 'lambdatest', buckets: 1 ],
+        [ browser: 'chrome', provider: 'lambdatest', os: 'macOS Sonoma', buckets: 1 ],
+        [ browser: 'firefox', provider: 'lambdatest', os: 'macOS Sonoma', buckets: 1 ],
+        [ browser: 'safari', provider: 'lambdatest', os: 'macOS Sonoma', buckets: 1 ],
+      ],
+      prepareTests: {
+        yarnInstall()
+        sh 'yarn build'
+      },
+      testDirs: [ 'modules/sample/src/test/ts/**/pass' ],
+      custom: '--config modules/sample/tsconfig.json --customRoutes modules/sample/routes.json --polyfills Promise Symbol'
```

Comment:
Can we remove the polyfills now since we are running on browsers that support Promise and Symbol?

Reply:
I thought I tried that and it had some issues. But you're right it makes sense to wipe that out now!

Reply:
Turns out none of our editor automated tests use this anymore (a few manual still did). But it's still worth removing just in case!